run.sh

```diff
@@ -18,11 +18,17 @@
 set -e
 
 BASE=$(dirname $(readlink -f $0))
+CMD=$1
+if [ -z $2 ]; then
```

Comment:
Can we have this as a `-r` style argument, rather than literally any string? I'm fine with needing it to be first (so processing args is easy).